lib/elixir/lib/calendar.ex

```diff
@@ -151,10 +228,10 @@ defmodule Date do
       true
 
   """
-  @spec utc_today() :: t
-  def utc_today() do
-    {:ok, {year, month, day}, _, _} = Calendar.ISO.from_unix(:os.system_time, :native)
-    %Date{year: year, month: month, day: day}
+  @spec utc_today(Calendar.calendar) :: t
+  def utc_today(calendar \\ Calendar.ISO) do
```

Comment:
So basically: We can only truncate it to a Date or NaiveDateTime áfter the conversion from Calendar.ISO to SomeOtherCalendar, as DateTime-level of precision is needed here.